athulkj.md

```diff
@@ -0,0 +1 @@
+https://docs.google.com/document/d/1SpEOMeNag1INBWrWxqhajJrELvSbfd0p8cT15TiUOGk/edit?usp=sharing
```

Comment:
The target of the exercise is to help you learn markdown. Remove this link and update the PR with actual content in markdown format. Refer #1 for reference.